json_spec/output/diagnosis.json

```diff
@@ -23,7 +23,11 @@
         "subcomponent": {
             "description": "TODO: link to md",
             "$ref": "/opencomputeproject/ocp-diag-core/subcomponent"
-        }
+        },
+	"sourceLocation": {
+            "description": "TODO: link to md doc",
```

Comment:
you can actually fill these in now; the TODO was before the spec README.md was available
it should be a "short string to describe the item + link to md doc with #anchor"

Reply:
updated